test/integration/scheduler/volume_binding_test.go

```diff
@@ -361,37 +459,61 @@ func TestPVAffinityConflict(t *testing.T) {
 	}
 }
 
-func setupCluster(t *testing.T, nsName string, numberOfNodes int) *testConfig {
-	// Enable feature gates
-	utilfeature.DefaultFeatureGate.Set("VolumeScheduling=true,PersistentLocalVolumes=true")
+func setupCluster(t *testing.T, nsName string, numberOfNodes int, features map[string]bool) *testConfig {
+	oldFeatures := make(map[string]bool, len(features))
+	for feature := range features {
+		oldFeatures[feature] = utilfeature.DefaultFeatureGate.Enabled(utilfeature.Feature(feature))
+	}
+	// Set feature gates
+	utilfeature.DefaultFeatureGate.SetFromMap(features)
 
 	controllerCh := make(chan struct{})
 
-	context := initTestScheduler(t, initTestMaster(t, nsName, nil), controllerCh, false, nil)
+	context := initTestSchedulerWithOptions(t, initTestMaster(t, nsName, nil), controllerCh, false, nil, false, 0)
```

Comment:
the resync period of 1 second actually uncovered a bug in the volume binder.  It may be good to make resync configurable here too

Reply:
Like this? https://github.com/kubernetes/kubernetes/pull/65616/commits/0c25d57daccf85363d7aec98f39a21bb9d9aa8d8